src/Directory.Build.rsp

```diff
@@ -0,0 +1,4 @@
+-nologo
+-nr:false
+-v:m
```

Comment:
1. When are these used?
2. Isn't -nologo redundant? It seems to be specified everywhere already.
3. How do you get the binlogs from the CI server?

Reply:
1. Every build.
2. There is a bug open in MSBuild to respect `-nologo` in the response files. So for now it has to be added everywhere.
3. The "Summary" of the build shows all the uploaded artifacts. `Logs_#####` is one of them. You can "Open" that then double click the .binlogs inside if you have binlog viewer installed. Better experience than I expected.

Reply:
I feel like we need binlogs for everything if everything is building with minimal verbosity.

Reply:
We need binlogs for all builds.